tests/reducers/labOrderReducers/labOrderableReducer.test.js

```diff
@@ -0,0 +1,44 @@
+import {
+  fetchLabOrderablesSuccess,
+  fetchLabOrderablesFailure
+} from '../../../app/js/actions/labOrders/labOrderableAction';
+import labOrderableReducer from '../../../app/js/reducers/labOrders/labOrderableReducer';
+
+describe('Lab order reducer', () => {
+  let initialState;
+  beforeEach(() => {
+    initialState = {
+      labOrderables: [{}, {}, {}],
+      loading: false,
+      error: '',
+    };
+  });
+
+  it('should handle `FETCH_LAB_ORDERABLE_SUCCESS`', () => {
+    const orderable = {
+      labOrderables: [{ id: 1, display: 'outpatient' }],
+    };
+    const expectedState = {
+      labOrderables: [{ id: 1, display: 'outpatient' }],
+      loading: false,
+      error: '',
+    };
+    const actualState = labOrderableReducer(initialState, fetchLabOrderablesSuccess(orderable));
+    expect(actualState).toEqual(expectedState);
```

Comment:
Nice job with adding tests to your implementations 👍 . 
However, one thing to note is that this is a unit test and as a result, you should be testing just this reducer alone, you have to reduce the use of external codes as much as possible. 
I see you are importing the actual actions in this reducer test, what if the actions themselves are not properly written(and they contain bugs), this test would fail and that would make it seem like the reducer has bugs not knowing its the actions.
I suggest you do something like this: 
```
const mockAction = ... // mock the action
const actualState = labOrderableReducer(initialState, mockAction);
```

Reply:
Done